js/dropdown-autoflip.js

```diff
@@ -97,26 +97,27 @@
 	}
 
 	function _getContainer(element) {
-		var containerElement, isWindow;
+		// default to window as container
+		var containerElement = window;
+		var isWindow = true;
 
-		// manual override
-		if (element.attr('data-target')) {
-			containerElement = element.attr('data-target');
-			isWindow = false;
-		} else {
-			// default to window otherwise
-			containerElement = window;
-			isWindow = true;
-
-			// unless there's a parent element with non-visible overflow
-			$.each(element.parents(), function (index, value) {
-				if ($(value).css('overflow') !== 'visible') {
-					containerElement = value;
+		// parse decision tree
+		var targetSelector = element.attr('data-target') || false;
+		var nonWindowTargetChosen = targetSelector !== 'window' && targetSelector;
+		var noTargetChosen = targetSelector !== 'window' && !targetSelector;
+
+		if (nonWindowTargetChosen) {
+			targetedElement = $(targetSelector);
```

Comment:
I was just going to comment on this... @interactivellama, when I run `grunt test` jshint catches this and says it is undefined